addons/rook-ceph/enable

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import subprocess
+
+import click
+
+SNAP = pathlib.Path(os.getenv("SNAP") or "/snap/microk8s/current")
+KUBECTL = SNAP / "microk8s-kubectl.wrapper"
+
+
+def clone_repository(repo_url, destination_dir, branch):
+    """
+    Clone a particular branch of a git repository to a destination directory.
+
+    :param repo_url: The repository URL.
+    :param destination_dir: The destination directory.
+    :param branch: The branch to clone.
+    """
+    click.echo(f"Cloning repository version {branch} in {destination_dir}")
+    try:
+        subprocess.call("git clone {} {} -b {}".format(repo_url, destination_dir, branch).split())
```

Comment:
You do not know the user has git installed. You should get a release artifact or even better deploy a helm package.

Reply:
Using Helm won't allow us to control the rook version. A release artifact sounds like a better approach.